csi.proto

```diff
@@ -565,6 +600,11 @@ message NodePublishVolumeRequest {
   // Whether to publish the volume in readonly mode. This field is
   // REQUIRED.
   bool readonly = 7;
+
+  // End user credentials used to authenticate/authorize node publish request.
+  // These Credentials are not passed in on NodeUnpublishVolume calls.
```

Comment:
Why the "These Credentials are not passed in on NodeUnpublishVolume calls." statement? It's confusing because the controller unpublish call DOES have Credentials now

Reply:
Whoops. Good catch. Forgot to delete that.

Reply:
Done